CONTRIBUTING.md

```diff
@@ -0,0 +1,80 @@
+# Contributing to Hydroflow
+
+Thanks for your interesting in contributing to Hydroflow! This is an experimental, reserach-driven
```

Comment:
Minor typo.

`s/reserach/research/`

Reply:
`s/interesting/interest`

Reply:
Hmm I should probably enable spell check 😅